dog.py

```diff
@@ -18,19 +18,15 @@
 MAX_DOG_CONFIG_VERSION = 2
 
 # Constants for consistent naming of dog variables, etc.
+# Configuration keys (documented, external)
```

Comment:
```suggestion
# Configuration keys (documented, externally in Configuration.md)
```

Reply:
I think we should refer to the .md file with the documentation.

Reply:
Done

Reply:
Cool, then I am very happy to merge it.